torchvision/models/detection/rpn.py

```diff
@@ -73,7 +73,7 @@ def generate_anchors(scales, aspect_ratios, dtype=torch.float32, device="cpu"):
         hs = (h_ratios[:, None] * scales[None, :]).view(-1)
 
         base_anchors = torch.stack([-ws, -hs, ws, hs], dim=1) / 2
-        return base_anchors.round()
+        return base_anchors
```

Comment:
for my own curiosity; is this commented because it's a `_local_scalar_dense` call?

Reply:
`aten::round` more specifically